Makefile

```diff
@@ -47,6 +49,14 @@ endif
 ci-exec:
 	docker exec -i -t $(VERACRUZ_CONTAINER)_ci_$(USER) /bin/bash
 
+.PHONY:
+nitro-run: nitro-base
+	docker run --privileged -d -v $(abspath $(VERACRUZ_ROOT)):/work/veracruz -v $(HOME)/.cargo/registry/:/usr/local/cargo/registry/ -v /usr/bin:/host/bin -v /usr/share/nitro_enclaves:/usr/share/nitro_enclaves -v /run/nitro_enclaves:/run/nitro_enclaves -v /etc/nitro_enclaves:/etc/nitro_enclaves --device=/dev/vsock:/dev/vsock -v /var/run/docker.sock:/var/run/docker.sock --device=/dev/nitro_enclaves:/dev/nitro_enclaves --env TABASCO_IP_ADDRESS=$(LOCALIP) -p $(LOCALIP):3010:3010/tcp --name $(VERACRUZ_CONTAINER)_nitro_$(USER) $(VERACRUZ_DOCKER_IMAGE)_nitro:$(USER) sleep inf
```

Comment:
A stray mention of Tabasco in `TABASCO_IP_ADDRESS`.

Reply:
Is that even needed any more?

Reply:
Yes, still needed until #180 is merged. The name is anachronistic, but exists in current code on main until the #180 merge.

Reply:
OK, cool.